### PR TITLE
Add SQLite

### PR DESCRIPTION
**Pull request by cLupus**

### CODE REVIEW

**Comment by Stratus3D** [comment]
Thanks! This looks good @cLupus! One question I did have when looking at your plugin:

```
script: asdf plugin-test sqlite $TRAVIS_BUILD_DIR 'node --version'
```

Is `node --version` the actual test command you want to run? Wouldn't you want to run something like `sqlite3 --version`?

File: plugins/sqlite [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/cLupus/asdf-sqlite
```
Comment (Stratus3D) [inline plugins/sqlite]: Actually, now that I think about it, I think this URL needs `.git` at the end.

**Comment by cLupus** [comment]
It was supposed to be `sqlite3 --version`. I'll fix it.

*Side note*: Is there a precedent for what to do with the various output?

**Comment by Stratus3D** [comment]
I don't think it's used by the `plugin-test` code right now. I think it's just to verify the command itself succeeded.

File: plugins/sqlite (hunk 1 above)
Comment (Stratus3D) [reply plugins/sqlite]: Either is fine. If you discover an issue just create a PR and I'll merge it.

**Pull request merged**